tests/unit/mixins/sl-tooltip-enabled-test.js

```diff
@@ -156,9 +156,9 @@ test( 'enablePopover() - Renders popover', function( assert ) {
     subject.enablePopover();
 
     assert.equal(
-        subject.get( 'data-toggle' ),
+        subject.get( 'dataToggle' ),
         'popover',
-        '"data-toggle" has correct value'
+        '"dataToggle" has correct value'
     );
 
```

Comment:
Add a test for `dataToggle to ensure that default value is `null` and doesn't get changed to something else, like `focus`.